ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -189,27 +189,15 @@ public void handleGet(final RestRequest restRequest, final RestResponseChannel r
         GetCallback routerCallback = new GetCallback(restRequest, restResponseChannel, subresource, options);
         SecurityProcessRequestCallback securityCallback =
             new SecurityProcessRequestCallback(restRequest, restResponseChannel, routerCallback);
-        RestRequestMetrics requestMetrics =
-            restRequest.getSSLSession() != null ? frontendMetrics.getBlobSSLMetrics : frontendMetrics.getBlobMetrics;
+        boolean isSsl = restRequest.getSSLSession() != null;
         if (subresource != null) {
           logger.trace("Sub-resource requested: {}", subresource);
-          switch (subresource) {
-            case BlobInfo:
-              requestMetrics = restRequest.getSSLSession() != null ? frontendMetrics.getBlobInfoSSLMetrics
-                  : frontendMetrics.getBlobInfoMetrics;
-              break;
-            case UserMetadata:
-              requestMetrics = restRequest.getSSLSession() != null ? frontendMetrics.getUserMetadataSSLMetrics
-                  : frontendMetrics.getUserMetadataMetrics;
-              break;
-            case Replicas:
-              requestMetrics = restRequest.getSSLSession() != null ? frontendMetrics.getReplicasSSLMetrics
-                  : frontendMetrics.getReplicasMetrics;
-              securityCallback = new SecurityProcessRequestCallback(restRequest, restResponseChannel);
-              break;
+          if (subresource == SubResource.Replicas) {
+            securityCallback = new SecurityProcessRequestCallback(restRequest, restResponseChannel);
           }
         }
-        restRequest.getMetricsTracker().injectMetrics(requestMetrics);
+        restRequest.getMetricsTracker()
+            .injectMetrics(getRestRequestMetricsForGET(frontendMetrics, subresource, isSsl, false));
```

Comment:
`..Get` - just like `handleGet()`. Coding style says that three letter words have to be camel case